README.rst

```diff
@@ -50,23 +50,23 @@ automation solutions can be built on top of its building blocks.
 Other popular building blocks that are part of the OpenWISP ecosystem are:
 
 - `openwisp-controller
-  <https://github.com/openwisp/openwisp-controller>`_: network and WiFi
+  <https://openwisp.io/docs/__new__/dev/controller/>`_: network and WiFi
```

Comment:
Please set the URL without the `__new__` prefix so once we're ready we can just merge it without further editing.